src/bfloat16.adoc

```diff
@@ -658,7 +658,7 @@ However, here the SEW-width format is limited to BF16.
 
 [NOTE]
 ====
-If the input is normal or infinity, the BF16 encoded value is shifted
+If the input is normal or {inf}, the BF16 encoded value is shifted
```

Comment:
Undo this one; the intent was for the word infinity to be spelled out

Reply:
Looks like there were two of these and I didn't catch the other, but I'll handle it myself as part of merging this.